packages/amplify_datastore/example/ios/unit_tests/AmplifySerializedModelUnitTests.swift

```diff
@@ -118,4 +145,76 @@ class AmplifySerializedModelUnitTests: XCTestCase {
         XCTAssertEqual(ourSd["timeType"] as! String , refSd["timeType"] as! String)
         XCTAssertEqual(ourSd["enumType"] as! String , refSd["enumType"] as! String)
     }
+
+    func test_model_nests_custom_type_serialization() throws {
```

Comment:
Nit: Not sure if this is similar to some of the other changes made
```suggestion
    func test_model_nested_custom_type_serialization() throws {
```